schema/partner_artist.js

```diff
@@ -61,3 +62,7 @@ const PartnerArtist = {
 }
 
 export default PartnerArtist
+
+export const partnerArtistConnection = connectionDefinitions({
+  nodeType: PartnerArtistType,
```

Comment:
Wasn’t the original goal to make ‘partner artist’ an edge, rather than a node?

Reply:
Ah yes, good point!